src/helpers/flare.ts

```diff
@@ -61,7 +61,16 @@ export const sendToDatadog = async (
       const errResponse: string = (err.response?.data.error as string) ?? ''
       const errorMessage = err.message ?? ''
 
-      throw Error(`Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`)
+      let message = `Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`
+      const code = err.response?.status
+      // The error message doesn't say why there was an error, so it's important to tell the user why the request failed.
+      if (code === 500) {
+        message += 'Are your case ID and email correct?\n'
+      } else if (code === 403) {
+        message += 'Is your Datadog API key correct?\n'
+      }
```

Comment:
Prefer a switch. 403 is indeed bad auth. 400 is there too, and 500 as well. Sent you the endpoint rejects privately.

Reply:
Thanks, added 400 and converted to switch statements.